fiftyone/core/singletons.py

```
@@ -24,7 +30,16 @@ def __new__(metacls, *args, **kwargs):
         cls._instances = weakref.WeakValueDictionary()
         return cls
 
-    def __call__(cls, name=None, _create=True, *args, **kwargs):
+    def _make_instance(cls, name, _create, *args, **kwargs):
+        instance = cls.__new__(cls)
+        instance.__init__(name=name, _create=_create, *args, **kwargs)
```

Ruff (0.12.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)